### PR TITLE
chore(deps): fix dependencies conflicts

### PR DESCRIPTION
**Pull request by kunyan**
Since the `react-scripts` has been upgraded to v4
There are some conflicts like `eslint`, `jest` with the root package.json

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by kunyan** [comment]
@mwcz  @sayak-sarkar 

I still suggest we should move `manager` and `documentation` out of monorepo

for `manager`, it is using `react-scripts`, which included many dependencies and will conflict with our root package.json setting. now, we can not easily upgrade the `react-scripts` or the `eslint`, `jest` in root package.json

for documentation. it's not created by us. we only create markdown files. so we should not put it into the `packages`
it also has some dependencies issue

**Comment by kunyan** [comment]
Both of them has nothing to share with other packages.

File: packages/documentation/package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "test": "docusaurus build"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.65",
-    "@docusaurus/preset-classic": "2.0.0-alpha.65",
+    "@docusaurus/core": "2.0.0-alpha.66",
+    "@docusaurus/preset-classic": "2.0.0-alpha.66",
```
Comment (sayak-sarkar) [inline packages/documentation/package.json:16]: There are some issues with this version of Docusaurus which is why @mwcz rolled back this version in his last PR.

**Review by sayak-sarkar — CHANGES_REQUESTED** [review]

**Comment by sayak-sarkar** [comment]
Instead of breaking up the monorepo, we can simply configure lerna and renovate to maintain separate dependency versions for individual packages. That way dependency updates to one package wont impact the other one.

**Comment by mwcz** [comment]
I agree with @sayak-sarkar, we shouldn't break up the monorepo.  I've said a million times that the ability to clone the project, `npm install`, and `npm start` to get a working environment is critical.  I'm going to keep saying it until we get there.  Keeping the monorepo together is essential.

If we pull manager and docs out of the monorepo, here's what will happen:

 - documentation will be updated out of sync with the actual features landing, if it gets updated at all
 - changes to the API will not be well tested with manager
 - onboarding new contributors will be more difficult, requiring more documentation, and more fielding of questions
 - version numbers of manager and the rest of SPAship will be out of sync

Solution:

If some devDependencies in the monorepo differ from what's in some packages, move the conflicting root packages into the _other_ packages' dep tree.  Ie, remove eslint from root/package.json and put it into the package.json's for every package that needs that version.  Renovate will take care of keeping the versions up to date, so there's no extra burden on us.

**Pull request closed without merging**